src/Native/LibTorchSharp/THSTensor.cpp

```diff
@@ -48,7 +48,7 @@ Tensor THSTensor_aminmax(const Tensor tensor, const int64_t dim, bool keepdim, T
 {
     std::tuple<at::Tensor, at::Tensor> res;
     CATCH(res = dim == -1 ? tensor->aminmax(c10::nullopt, keepdim) : tensor->aminmax(dim, keepdim);)
-    *max = ResultTensor(std::get<1>(res));
+        * max = ResultTensor(std::get<1>(res));
```

Comment:
>  [](http://example.com/codeflow?start=9&length=1)

nit: remove this space.

Reply:
Sure. BTW, Ctrl-K,Ctrl-D does such a poor job in these C++ files with the CATCH() macro, that I have generally given up on the indentation. This isn't one of those, I don't think, but I thought I'd mention it.